src/platform/Darwin/BleConnectionDelegateImpl.mm

```diff
@@ -575,7 +574,11 @@ - (void)stopScanning
     if (!_centralManager) {
         return;
```

Comment:
Should we do this even if we have a timer?  Or can we not have a timer in this case?

Reply:
It should not happens.